core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -4023,6 +4023,27 @@ private void checkLiteral2(String expression, String expected) {
     sql(query).ok(expected);
   }
 
+  @Test public void testJsonInsert() {
+    String query = "select json_insert(\"product_name\", '$', 10) from \"product\"";
+    final String expected = "SELECT JSON_INSERT(\"product_name\", '$', 10)\n"
+        + "FROM \"foodmart\".\"product\"";
```

Comment:
For all the tests, add 2 more kind of value test:

1. null values
2. character that needs to encode/decodes e.g. "\n \" "\t" "\r" ..

Reply:
1. Do not test the 'null' and "\n" in one query
2. put each query and expected pair together instead of putting all the queries together.